Misc/NEWS.d/next/Library/2025-12-28-12-38-53.gh-issue-143241.yW250u.rst

```diff
@@ -0,0 +1 @@
+Fix infinite loop in zoneinfo._common.load_data caused by a malformed TZif file.
```

Comment:
I think we don't need to mention `load_data` directly, but we could mention the user-facing consequences (thatis, mention a public API that would use that function). In addition, we should document (if it's not already the case) that a ValueError is raised (and not an EOFError). If other ValueError are documented in the RST docs, then it's good to indicate that truncated files will also raise such exception.